query_test.go

```diff
@@ -138,18 +138,18 @@ func TestInsertSliceStructPtr(t *testing.T) {
 	now = time.Now()
 
 	data := []*testRow{
-		&testRow{
+		{
```

Comment:
Cool, didnt know you can do that!

Reply:
I suggest you use modern IDEs and add lint tests (https://golangci.com/) to your codes.